docs/content_management/create_edit_pages.md

```diff
@@ -77,33 +96,72 @@ Each block is unique in its function and enables you to customize the content ap
+
+![Block properties](block_properties.png)
+
+This is a default behavior. You can globally turn off automatic opening of the block settings panel in the user settings.
+First, access your user settings on the right side of the top bar:
+
+![Elements menu](img/user_settings.png "User settings")
+
+Then, go to **My preferences** tab, **Edit** section.
+Here, you can find `Automatically open block settings in builder` setting, which, by default, is set up to `enabled` value.
+To change this behavior, click on **Edit** icon, find the setting, and change its value to `disabled`.
+
+![User settings - blocks](img/user_settings_blocks.png "User settings - blocks settings")
 
 [[= product_name =]] comes with a set of ready-to-use Page blocks,
 but [developers can add their own]([[= developer_doc =]]/content_management/pages/create_custom_page_block/).
 
-![Elements menu](img/elements_menu.png "Elements menu")
-
 For a list of blocks available out of the box, see [Block reference](block_reference.md).
 
-!!! note 
+!!! note
 
-    Before you add a block that involves products, product types, or product categories, make 
-    sure your that your [user Role](../permission_management/permissions_and_users.md) has 
+    Before you add a block that involves products, product types, or product categories, make
+    sure your that your [user Role](../permission_management/permissions_and_users.md) has
     the `Product/View` and `Product type/View` permission.
 
     Before you add a block that uses results derived from a [Personalization](../personalization/personalization.md) scenario, 
-    for example, [Dynamic targeting](block_reference.md#dynamic-targeting-block) or [Recently added block](block_reference.md#recently-added-block), make sure that the scenario is 
-    [properly configured](../personalization/configure_scenarios.md).
+    for example, [Dynamic targeting](block_reference.md#dynamic-targeting-block) or [Recently added block](block_reference.md#recently-added-block), make sure that the scenario is [properly configured](../personalization/configure_scenarios.md).
+
+You can work with the block, for example, duplicate it, change its position or delete it.
+To access options menu, hover over the block - the toolbar appears.
+Click the icon to see more options.
+
+![Block toolbar](img/block_settings_menu.png "Block toolbar")
+
+Available options are:
+
+- Move up block - allows you to change position of the block on the page by moving it up
+- Move down block - allows you to change position of the block on the page by moving it down
+- Block settings - allows you to access properties window
+- Duplicate - duplicates existing block with its settings by creating its copy
+- Refresh - refreshes preview of the block
+- Delete - deletes existing block
+
+For your comfort, you can adjust the size of the block settings window. When you hover over the window frame, blue line appears. 
```

Comment:
```suggestion
For your comfort, you can adjust the size of the block settings window. 
When you hover over the window frame, blue line appears. 
```

Also, I feel like this paragrapt belongs in the next section, it could be under the screenshot, right before the "Settings available for blocks are divided into..." paragraph.